### PR TITLE
233 feature ansible role for deploying hipstershop

### LINKED ISSUE #233: [FEATURE] Ansible role for deploying Hipstershop
Is your feature request related to a problem? Please describe.
Right now, app deployment often happens as part of the use case (e.g. simplenode service for demos, unguard for appsec hot, easytravel for IM HOT). As part of building use cases, users of the acebox platform might want demo apps automatically deployed and not having to take care of it separately.

Describe the solution you'd like
At least the following ansible roles:
- Hipstershop

Other roles, like use case roles, can then reuse these

### PR DESCRIPTION
**Pull request by eemrdog**

### CODE REVIEW

File: user-skel/ansible/roles/app-hipstershop/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    dest: "{{ role_path }}/files/kustomize/base/patches/ingress.yaml"
+    content: |
+      apiVersion: networking.k8s.io/v1
+      kind: Ingress
+      metadata:
+        name: frontend-ingress
+      spec:
+        ingressClassName: "{{ ingress_class }}"
+        rules:
+        - host: "{{ ingress_domain }}"
```
Comment (kristofre) [inline user-skel/ansible/roles/app-hipstershop/tasks/main.yml]: should this not be a different variable indicating that it is a subdomain of `ingress_domain`?

File: user-skel/ansible/roles/app-hipstershop/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+# app deployment parameters
+hipstershop_namespace: "hipstershop"
+hipstershop_image_tag: "215"
+
```
Comment (kristofre) [inline user-skel/ansible/roles/app-hipstershop/defaults/main.yml:6]: we should have a variable for the ingress

File: user-skel/ansible/roles/app-hipstershop/tasks/main.yml (hunk 1 above)
Comment (eemrdog) [reply user-skel/ansible/roles/app-hipstershop/tasks/main.yml]: I resolved this issue. By the way, I realized that unguard has the same one, so I will create a bug and resolve that one too

File: user-skel/ansible/roles/app-hipstershop/defaults/main.yml (hunk 2 above)
Comment (eemrdog) [reply user-skel/ansible/roles/app-hipstershop/defaults/main.yml:6]: done

**Review by kristofre — APPROVED** [review]

**Pull request merged**